core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -2886,8 +2888,7 @@ public static <E extends Comparable> Enumerable<FlatLists.ComparableList<E>> pro
   /** Adds a given number of months to a timestamp, represented as the number
    * of milliseconds since the epoch. */
   public static long addMonths(long timestamp, int m) {
-    final long millis =
-        DateTimeUtils.floorMod(timestamp, DateTimeUtils.MILLIS_PER_DAY);
+    final long millis = floorMod(timestamp, DateTimeUtils.MILLIS_PER_DAY);
```

Comment:
These changes were done to try fix the Github action failures due to deprecated code. Unfortunately a new issue arose afterwards that requires merging more Calcite changes to support.

Reply:
I don't see any github issues here, so is this resolved?